docs/_tutorials/bert-pretraining.md

```diff
@@ -0,0 +1,319 @@
+---
+title: "Pre-training Bing BERT"
+excerpt: ""
+---
+
+In this tutorial we will apply DeepSpeed to pre-train the BERT
+(**B**idirectional **E**ncoder **R**epresentations from **T**ransformers),
+which is widely used for many Natural Language Processing (NLP) tasks. The
+details of BERT can be found here: [BERT: Pre-training of Deep Bidirectional
+Transformers for Language Understanding](https://arxiv.org/abs/1810.04805).
+
+
+We will go through how to setup the data pipeline and how to run the original
+BERT model. Then we will show step-by-step how to modify the model to
+leverage DeepSpeed. Finally, we demonstrate the performance evaluation and
+memory usage reduction from using DeepSpeed.
+
+## Pre-training Bing BERT without DeepSpeed
+
+We work from a adaptations of
+[huggingface/transformers](https://github.com/huggingface/transformers) and
+NVIDIA. We have forked this repo under
```

Comment:
I'd change this to "We work from adaptations of ...", removing 'a'.

Also I'd add a link for nvidia to this repo: https://github.com/NVIDIA/DeepLearningExamples

Reply:
Done